_includes/navigation.html

```diff
@@ -20,25 +20,20 @@
           <a href="/">Home</a>
         </li>
 
-        <li class="dropdown hidden-sm {% if page.section == 'getting-started' %}active{% endif %}">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Getting started<span class="caret"></span></a>
-          <ul class="dropdown-menu" role="menu">
-            <li><a href="/getting-started/overview/">Overview</a></li>
-            <li><a href="/getting-started/installation-and-configuration/">Installation &amp; Configuration</a></li>
-            <li><a href="/getting-started/creating-mobile-apps/">Creating Mobile Apps</a></li>
-            <li><a href="/getting-started/enabling-mobile-services/">Enabling Mobile Services</a></li>  
-          </ul>
+        <li class="{% if page.section == 'getting-started' %}active{% endif %}">
+          <a href="/getting-started/overview/">Getting Started</a>
         </li>
 
 
         <li class="dropdown {% if page.section == 'mobile-services' %}active{% endif %}">
           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Mobile Services<span class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
-              <li><a href="/services/push/">AeroGear <strong>Push</strong></a></li>
-              <li><a href="/services/mobile-ci-cd/">AeroGear <strong>Mobile CI/CD</strong></a></li>
-              <li><a href="/services/metrics/">AeroGear <strong>Metrics</strong></a></li>
-              <li><a href="/services/crc/">AeroGear <strong>CRC</strong></a></li>
-              <li><a href="/services/keycloak/">AeroGear <strong>Keycloak</strong></a></li>
+              <li><a href="/services/mobile-ci-cd/">Mobile CI/CD</a></li>
+              <li><a href="/services/push/">Push Notifications</a></li>
+              <li><a href="/services/metrics/">Metrics</a></li>
+              <li><a href="/services/crc/">Runtime Connectors</a></li>
```

Comment:
Are we calling this `Runtime Connectors`? On @dimitraz landing page changes, we're calling this `Custom Runtime Connector`. I think it would be best to stick to one name for consisitency?

Reply:
@dimitraz @JameelB yeah it's looking like there's going to be a name change here to "Runtime Connector" instead. I can tackle renaming as well.